tools/cli/admin.go

```diff
@@ -811,6 +811,20 @@ func newAdminClusterCommands() []cli.Command {
 				AdminDescribeCluster(c)
 			},
 		},
+		{
+			Name:    "failover",
+			Aliases: []string{"fo"},
+			Usage:   "Failover domains with domain data IsManagedByCadence=true to target cluster",
+			Flags: []cli.Flag{
```

Comment:
It seems like this command should only be executed by cadence team. Should the command also takes in a security token and verify its content before actually execution?

Just an idea to explore. I understand this is not easy as the underlying domain failover command doesn't require security token.

Reply:
Good point. But security token on open sourced repo is merely support on server API. 
Internally we will be protected by the auth feature.